### ISSUE TITLE
The meaning of "SPN" is nowhere explained

### ISSUE THREAD

**Issue by privacytoolsIO** [safing/spn#149]
Hey, the word "SPN" is nowhere explained on the safing.io website. It's just explained in the GitHub browser title. Currently adding SPN to: https://www.privacytools.io/dvpn

**Comment by Raphty** [comment]
Yes, we started to get this comment a lot more in recent weeks. Thanks for raising the issue, and also thanks for the listing. 💙

A comment to the listing though, you can configure exits per app, but tunnels are built by domain, so you have multiple tunnels per app.

**Comment by davegson** [comment]
I wanted to leave the issue open until the texts are improved on the website - will probably do so today!
